apcd-cms/src/apps/view_users/views.py

```diff
@@ -64,14 +64,14 @@ def _set_user(usr):
                     'user_id': usr[1],
                     'user_email': usr[2],
                     'user_name': usr[3],
-                    'org_name': usr[4],
+                    'entity_name': usr[4] if usr[4] else "Not Applicable",
                     'created_at': usr[5],
                     'updated_at': usr[6],
                     'notes': usr[7],
                     'status': 'Active' if usr[8] else 'Inactive',
                     'user_number': usr[9],
                     'role_name': usr[10],
-                    'org_name_no_parens': usr[4].replace("(", "").replace(")", ""),  # just for filtering purposes
+                    'entity_name_no_parens': usr[4].replace("(", "").replace(")", "") if usr[4] else "Not Applicable",  # just for filtering purposes
                     'active': usr[8],
                 }
 
```

Comment:
For line 78: 
`context['header'] = ['User ID', 'Name', 'Entity Organization', 'Role', 'Status', 'User Number', 'See More']`

To match other admin tables and be clear which field this is pulling

Reply:
Changed to Entity Organization